cpp/tests/io/experimental/hybrid_scan_filters_test.cpp

```diff
@@ -716,11 +716,14 @@ TYPED_TEST(RowGroupFilteringWithDictTest, FilterFewLiteralsTyped)
   srand(0xace);
   using T = TypeParam;
 
-  auto constexpr num_concat = 1;
+  auto constexpr num_concat          = 1;
+  auto constexpr is_constant_strings = true;
+  auto constexpr is_nullable         = true;
 
-  // Specifying ZSTD compression to explicitly test decompression of dictionary pages
+  // Specifying SNAPPY compression to explicitly test decompression of dictionary pages
   auto const buffer =
-    std::get<1>(create_parquet_with_stats<T, num_concat>(100, cudf::io::compression_type::ZSTD));
+    std::get<1>(create_parquet_with_stats<T, num_concat, is_constant_strings, is_nullable>(
```

Comment:
Temporarily changed to avoid any failures from the recently reported ZSTD bug in nvcomp The bug doesn't show up here so I don't have strong feelings about keeping this change either so happy to revert if needed